infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -537,17 +536,13 @@ public void resetServer(@Nonnull RequestMsg req,
 
             for (Map.Entry<UUID, StreamAddressSpace> streamAddressSpace :
                     streamsAddressMap.entrySet()) {
-                log.info("Stream[{}] set to last trimmed address {} and {} addresses in the range" +
-                                " [{}-{}], on sequencer reset.",
+                log.info("Stream[{}]={} on sequencer reset.",
                         Utils.toReadableId(streamAddressSpace.getKey()),
-                        streamAddressSpace.getValue().getTrimMark(),
-                        streamAddressSpace.getValue().getAddressMap().getLongCardinality(),
-                        streamAddressSpace.getValue().getLowestAddress(),
-                        streamAddressSpace.getValue().getHighestAddress());
+                        streamAddressSpace.getValue());
                 if (log.isTraceEnabled()) {
                     log.trace("Stream[{}] address map on sequencer reset: {}",
                             Utils.toReadableId(streamAddressSpace.getKey()),
-                            streamAddressSpace.getValue().getAddressMap());
+                            streamAddressSpace.getValue().toString());
```

Comment:
this should be `streamAddressSpace.getValue().toArray()`

Reply:
That's fine, but it requires making `toArray()` public.